include/xrpl/protocol/TxMeta.h

```diff
@@ -128,13 +131,34 @@ class TxMeta
         return static_cast<bool>(mDelivered);
     }
 
+    void
+    setBatchId(uint256 const& batchId)
+    {
+        mBatchId = batchId;
+    }
+
+    uint256
+    getBatchId() const
```

Comment:
The field should be renamed `parentBatchId` everywhere instead of just `batchId`.